ibu-imager/installation_configuration_files/scripts/installation-configuration.sh

```diff
@@ -32,9 +32,9 @@ function recert {
     OLD_IP=$(cat /etc/default/seed-ip)
 
     ETCD_IMAGE="$(jq -r '.spec.containers[] | select(.name == "etcd") | .image' </etc/kubernetes/manifests/etcd-pod.yaml)"
-    RECERT_IMAGE="quay.io/edge-infrastructure/recert:latest"
+    local recert_tool_image=${RECERT_IMAGE}
```

Comment:
Maybe add a default value, so the script will be usable without environment file?

Reply:
Agree. 

Updated the script to use `quay.io/edge-infrastructure/recert:latest` image as the default value.